ckanext/unaids/react/jest/globals.js

```diff
@@ -29,3 +29,7 @@ global.$ = () => {
   // bootstrap modal actions
   return { modal: () => null }
 }
+
+global.console = () => {
+  return;
+}
```

Comment:
why do we need this?

Reply:
Yes, as there is some ugly logging due to mocking tests which we want to hide

```
$ yarn test
yarn run v1.22.17
warning package.json: No license field
$ jest
 PASS  components/BulkFileUploaderComponent/index.test.js
  ● Console

    console.error
      Mocked Promise Rejection

      33 |     const handleNetworkError = (errorType, error) => {
      34 |         setNetworkError(errorType);
    > 35 |         console.error(error);
         |                 ^
      36 |     };
      37 |
      38 |     const getAuthToken = () =>

      at handleNetworkError (components/BulkFileUploaderComponent/src/ModalBody.js:35:17)
      at components/BulkFileUploaderComponent/src/ModalBody.js:53:26

    console.error
      Mocked Promise Rejection

      33 |     const handleNetworkError = (errorType, error) => {
      34 |         setNetworkError(errorType);
    > 35 |         console.error(error);
         |                 ^
      36 |     };
      37 |
      38 |     const getAuthToken = () =>

      at handleNetworkError (components/BulkFileUploaderComponent/src/ModalBody.js:35:17)
      at components/BulkFileUploaderComponent/src/ModalBody.js:45:29

    console.error
      Mocked Promise Rejection

      33 |     const handleNetworkError = (errorType, error) => {
      34 |         setNetworkError(errorType);
    > 35 |         console.error(error);
         |                 ^
      36 |     };
      37 |
      38 |     const getAuthToken = () =>

      at handleNetworkError (components/BulkFileUploaderComponent/src/ModalBody.js:35:17)
      at components/BulkFileUploaderComponent/src/ModalBody.js:70:26

 PASS  components/FileInputComponent/index.test.js

Test Suites: 2 passed, 2 total
Tests:       27 passed, 27 total
Snapshots:   0 total
Time:        5.126 s
Ran all test suites.
Done in 8.31s.
```